libs/logging/src/log_event_ids.ts

```diff
@@ -555,6 +557,18 @@ const ExportLogFileFound: LogDetails = {
   documentationMessage:
     'When the user is exporting logs, indicates the success/failure of finding the expected log file on the machine.',
 };
+const LogConversionToCdfComplete: LogDetails = {
+  eventId: LogEventId.LogConversionToCdfComplete,
+  eventType: LogEventType.UserAction,
+  documentationMessage:
+    'The user has converted the log file to a CDF format for export. Success or failure indicated by disposition.',
+};
+const LogConversionToCdf: LogDetails = {
+  eventId: LogEventId.LogConversionToCdf,
+  eventType: LogEventType.UserAction,
+  documentationMessage:
+    'Message during the conversion of logs to the CDF format. If it is an error the disposition will be failure.',
```

Comment:
is this the start of CDF conversion? Maybe it should be more explicit if so?, at least in documentation message, possibly also in naming.